pkg/token/token.go

```diff
@@ -139,6 +139,14 @@ func NewSTSError(m string) STSError {
 	return STSError{message: m}
 }
 
+// STSThrottling is returned when there was STS Throttling.
+type STSThrottling struct {
+}
+
+func (e STSThrottling) Error() string {
+	return "STSThrottling"
```

Comment:
```suggestion
	return "sts getCallerIdentity was throttled: " + e.message
```

Reply:
Are `getCallerIdentity` the only calls that will go down this path?

Reply:
yes, it is